agent/src/capability/baseconfigstore.cpp

```diff
@@ -142,7 +143,7 @@ void BaseConfigStore::parseCapabilities(json_t *capabilities)
             throw CapabilityParseError(errorMessage);
         }
 
-        log_debug() << "Found capability \"" << capName << "\", parsing gateways...";
+        log_debug() << "Found capability \"" << capName << "";
```

Comment:
I guess you missed one " at the end
`log_debug() << "Found capability \"" << capName << "\"";`